docs/metrics.md

```diff
@@ -42,6 +41,29 @@ When we aggregate Lead time for changes we use the maximum of a period of time.
 | High                  | Between 1 day and 1 week     |
 | Elite                 | Less than 1 day              |
 
+### Sub-metrics
+
+There are several metrics that directly influence the lead time for changes,
+and can help you track in more detail what needs to be improved in your workflow.
+All the sub-metrics are aggregated by an interval (day, week or month) and have values for
+the average, median, and percentiles 5%, 20%, 25%, 75%, 80%, 95%.
+
+These metrics are extracted from pull requests,
+only include pull requests that have been merged (ignores closed without merge and currently open) and
+draft pull requests are treated as normal pull requests.
+
+#### Time to Open
+
+An approximation of the time a change takes to be developed.
+This metric assumes you are not developing after the PR is open.
+
+This metric is the difference between the pull request base commit author timestamp and the pull request created timestamp.
+
+#### Time to Review
+
+The time since a pull request is opened until it is merged.
+
+This metric is the difference between the the pull request merged timestamp and the pull request created timestamp.
```

Comment:
```suggestion
This metric is the difference between the pull request merged timestamp and the pull request created timestamp.
```

Reply:
Here it's the time between creating and **merging** the pull request.